tools/release_checks.rb

```diff
@@ -42,55 +41,34 @@ def check_tag
     end
   end
 
-  def check_jenkins_state(jenkins_url)
-    request = generate_request(jenkins_url)
-    json = JSON.parse(get_response(request))
-
-    actions = json["actions"].reject(&:empty?)
-    last_revision_tested?(get_last_revision, get_tested_revision(actions))
-
-    get_result(json["result"])
-  end
-
-  def get_response(request)
-    response = http.request(request)
-    if response.code != "200"
-      fail "Could not download Jenkins state information. Abort."
-    else
-      response.body
-    end
-  end
-
-  def generate_request(url)
+  def check_jenkins_state(url)
```

Comment:
Assignment Branch Condition size for check_jenkins_state is too high. [30.51/15]